### ISSUE TITLE
Unable to query shortcut for editor_menu

### ISSUE THREAD

**Issue by frankystone** [widelands/widelands#5716]
For some reason this error got not exposed to the terminal, so just a screen shot:

![missing_shortcut_for_Editor_menu](https://user-images.githubusercontent.com/6730556/208479369-8d389981-10d8-4b7d-8b2e-4766d046578a.jpg)

Looks like an Editor specific Problem, in a game all works fine.

**To Reproduce**
1. Open editor
2. Click on the help button
3. Click on Controls

**Expected behavior**
At least no error :-D

**Version:**
 - OS: archlinux
 - Widelands Version: git25986 (43599b4@master) Debug

**Additional context**
Not specific to this Problem but i remember such errors were formerly exposed to the terminal. Using a debug build with command line option `--verbose` just shows nothing related in the terminal.

**Comment by Noordfrees** [comment]
`data/scripting/editor/editor_controls.lua` and `data/tribes/scripting/help/controls.lua` must both be updated for all the new and changed shortcuts since #5705

**Comment by tothxa** [comment]
I broke it, I'll fix it. :)
